public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -2,13 +2,13 @@ block includes
   include ../_util-fns
   - var _iterableUrl = 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols';
   - var _boolean = 'truthy/falsey';
-  
+
 :marked
-  We typically display data in Angular by binding controls in an HTML template
+  Angular typically displays data by binding controls in an HTML template
```

Comment:
This changes the meaning. The dev specifies the binding, which enables the app to display property values. Can you think of another way to say this?